dataset-cellsystems-2021/fish/dataset.json

```diff
@@ -18,10 +18,20 @@
   "thumbnailRoot": "https://s3-us-west-2.amazonaws.com/bisque.allencell.org/cellsystems-2021-fish.1/Cell-Viewer_Thumbnails",
   "downloadRoot": "https://files.allencell.org/api/2.0/file/download?collection=cellsystems-2021-fish_1",
   "volumeViewerDataRoot": "https://s3-us-west-2.amazonaws.com/bisque.allencell.org/cellsystems-2021-fish.1/Cell-Viewer_Thumbnails",
-  "defaultXAxis": "cos",
-  "defaultYAxis": "myh7-density",
-  "defaultColorBy": "cell-age",
+  "xAxis": {
+    "default": "cos"
+  },
+  "yAxis": {
+    "default": "myh7-density"
+  },
+  "colorBy": {
+    "default": "cell-age"
+  },
+  "groupBy": {
+    "default": "rna-fish-targets"
+  },
   "featuresDisplayOrder": [
+    "rna-fish-targets",
```

Comment:
`featuresDataOrder` still makes sense in case the data is stored in a different order than in featureDefs.  but should  `featuresDisplayOrder` really be specific to each of xAxis, yAxis, groupBy and colorBy?  that would allow the order to be different for all of those.  or is `featuresDisplayOrder` going to be used for all 4 categories and always dictate the order?

Reply:
I think if someone requests that we can move this, right now I think it's fine for the display order to be the same for all of them, and in fact might bee confusing if they changed order.